tests/test_view_single_result.py

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+class TestSingleResult:
```

Comment:
I'm going to raise this as a separate issue (and mark it as suitable for Outreachy applicants), but we really don't need test classes in pytest. In unittest they're required because you inherit from a base `TestClass` that gives you helpers for assertions, as well as other things. In pytest you can just define test functions.